src/Microsoft.DotNet.Build.CloudTestTasks/AzureHelper.cs

```diff
@@ -0,0 +1,373 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AzureHelper.cs" company="">
+//   
+// </copyright>
+// <summary>
+//   The azure helper.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    #region
+
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Security.Cryptography;
+    using System.Text;
+    using System.Text.RegularExpressions;
+
+    #endregion
+
+    /// <summary>
+    /// The azure helper.
+    /// </summary>
+    public static class AzureHelper
+    {
+        /// <summary>
+        /// The storage api version.
+        /// </summary>
+        public static readonly string StorageApiVersion = "2015-04-05";
+
+        /// <summary>
+        /// The sas access type.
+        /// </summary>
+        public enum SasAccessType
+        {
+            /// <summary>
+            /// The read.
+            /// </summary>
+            read, 
+
+            /// <summary>
+            /// The write.
+            /// </summary>
+            write
+        };
+
+        /// <summary>
+        /// The authorization header.
```

Comment:
More doc-comment nits: The summary says what a method does; which means there should be a verb in it somewhere... it's a whole sentence.

```c#
/// <summary>
/// Computes the authorization header according to the current storage API version.
/// </summary>
```

Or something like that.